### ISSUE TITLE
Regression: changeDate event is broken for manual date input since 1.7.1

### ISSUE THREAD

**Issue by McNetic** [uxsolutions/bootstrap-datepicker#2395]
### Expected behaviour
When typing in the date input field and thus entering a date manually or changing the date, the changeDate event should be triggered

### Actual behaviour
This is a regression since version 1.6.1. At least in 1.7.1 and 1.8.0, the changeDate event is broken when manually entering/changing the date in the input field

### Datepicker version used

1.8.0

I have a #2376 ready to fix the issue. Unfortunately, I did not get feedback on it, so I assume I have to open a ticket.

**Comment by McNetic** [comment]
Could anyone please have a look at this?

**Comment by Ben-CA** [comment]
Is there any fix for this yet? With 1.8.0 with Chrome, this is an issue. There's numerous bug reports for this issue...

https://github.com/uxsolutions/bootstrap-datepicker/pull/2367
https://github.com/uxsolutions/bootstrap-datepicker/issues/1957
https://github.com/uxsolutions/bootstrap-datepicker/issues/2325

**Comment by julienGrd** [comment]
Hello guys, this issue is still active in 1.9.0 with chrome last version.

its a really bad bug, im surprise it was not solved in two years.

Is this library still active ?

**Comment by Ben-CA** [comment]
Agreed - I was surprised that 1.9 was released without this having been addressed. Would be nice to get a fix for this.

**Comment by julienGrd** [comment]
as a workaround guys you can listen to the input change event, it will work on both case (select date on the date picker or write directly)
````
var lObj = $('#' + id);
lObj.datepicker();
lObj.on('change', function (event, items) {
                                var lDate = lObj.datepicker('getUTCDate');
                                //do what you want with the date
                            });
````

But im still not sure if i want in my app a library with this kind of bug...

**Comment by toxpal** [comment]
Any update on this, please?
